demo/inferno-router-demo/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "inferno-router-demo",
+  "version": "8.0.5",
+  "description": "Influence CMS Demo",
+  "author": "Sebastian Ware <sebastian@urbantalk.se> (https://github.com/jhsware)",
+  "license": "SEE LICENSE IN LICENSE",
+  "source": "src/index.html",
+  "browserslist": "> 0.5%, last 2 versions, not dead",
+  "scripts": {
+    "dev": "npm-run-all -l --parallel  dev:*",
+    "dev:frontend": "FORCE_COLOR=1 parcel",
+    "dev:backend": "FORCE_COLOR=1 ts-node-dev --respawn src/server.ts"
+  },
+  "dependencies": {
+    "inferno": "file:../../packages/inferno",
+    "inferno-animation": "file:../../packages/inferno-animation",
+    "inferno-create-element": "file:../../packages/inferno-create-element",
+    "inferno-hydrate": "file:../../packages/inferno-hydrate",
+    "inferno-router": "file:../../packages/inferno-router",
+    "inferno-server": "file:../../packages/inferno-server",
+    "koa": "^2.14.1",
+    "koa-json-body": "^5.3.0",
+    "koa-logger": "^3.2.1",
+    "koa-mount": "^4.0.0",
+    "koa-router": "^12.0.0",
+    "koa-static": "^5.0.0"
+  },
+  "devDependencies": {
+    "@parcel/packager-ts": "^2.8.1",
+    "@babel/plugin-transform-modules-commonjs": "^7.19.6",
+    "@parcel/transformer-babel": "^2.8.1",
+    "@parcel/transformer-sass": "^2.8.1",
+    "@parcel/transformer-typescript-types": "^2.8.1",
+    "@types/node": "^18.11.9",
+    "babel-plugin-inferno": "^6.5.0",
```

Comment:
this example could use ts-plugin-inferno, no need for babel then

Reply:
https://github.com/infernojs/ts-plugin-inferno

Reply:
I looked at it but didn't understand how to configure ts-plugin-inferno for ParcelJS. Parcel uses SWC for transpilation of TS-code. If you have some pointers I would love to try it out.

Reply:
Ah ok